# Accession codes overview

Expression Atlas uses the [ArrayExpress](https://www.ebi.ac.uk/arrayexpress/) style for accessioning experiments. 
Each accession code startes with a capital `E` for "Experiment", followed by a dash, a four-letter-code, 
and ends with a dash and a running number. The four-letter-code relates to the original source of the data. 

Here is an overview of the letter codes used in Expression Atlas and their meaning, in alphabetical order.

### Accession codes of active projects/data sources

* **CURD**: Curated data, experiments from mixed sources or where only part of a study was used
* **ENAD**: European Nucleotide Archive (ENA) imported data
* **GEOD**: Gene Expression Omnibus (GEO) imported data
* **HCAD**: Human Cell Atlas (HCA) data from the HCA data coordination platform
* **MTAB**: ArrayExpress data submitted via Annotare
* **PROT**: Proteomics data from the PRIDE database

### Legacy accession codes that are not created any more

* **ATMX**: Arabidopsis experiments and array designs submitted through the At-MIAMExpress submission tool
* **CBIL**: Computational Biology and Informatics Laboratory at University of Pennsylvania
* **EHCA**: Human Cell Atlas (HCA) data from the first data release
* **ERAD**: European Read Archive data, pipeline submission from Wellcome Trust Sanger Institute
* **GEUV**: Genetic European Variation in Health and Disease (GEUVADIS), A European Medical Sequencing Consortium
* **JJRD**: Johnson & Johnson Pharmaceutical Research and Development
* **MAXD**: University of Manchester, Micorarray Group maxd software
* **MEXP**: ArrayExpress data submitted via the MIAMExpress webform submission tool (deprecated since July 2014)
* **NASC**: European Arabidopsis Stock Centre
* **TABM**: ArrayExpress data submitted via the tab2mage spreadsheet submission tool (deprecated since January 2012)
* **SYBR**: Sybaris Project
